gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -245,7 +246,8 @@ def get_config(
         "joint_faf": {
             "ht": get_combined_faf_release().ht(),
             "path": get_combined_faf_release().path,
-            "select": ["joint_freq", "joint_grpmax", "joint_faf", "joint_fafmax"],
+            "custom_select": custom_joint_faf_select,
+            "select": ["joint_freq", "joint_faf", "joint_fafmax"],
```

Comment:
Why do you make "custom_select" before "select" here?

Reply:
Another question: will the order change? Does it matter?

Reply:
I flipped it back but the order of the keys in the config shouldnt matter but I do agree that consistency is nice here. The annotation order will not change because it goes by the order in the `FINALIZED_SCHEMA` variable.